pkg/services/queryhistory/database.go

```diff
@@ -265,3 +265,56 @@ func (s QueryHistoryService) unstarQuery(ctx context.Context, user *models.Signe
 
 	return dto, nil
 }
+
+func (s QueryHistoryService) migrateQueries(ctx context.Context, user *models.SignedInUser, cmd MigrateQueriesToQueryHistoryCommand) error {
+	err := s.SQLStore.WithTransactionalDbSession(ctx, func(session *sqlstore.DBSession) error {
+		var instertError error
+
+		for _, query := range cmd.Queries {
+			// At the beginning of the loop check if there was any error and if so, break and return error
+			if instertError != nil {
+				break
+			}
```

Comment:
Why this is needed? Is it possible that `instertError` will be something other than `nil` at the beginning of the loop?
I think that we don't given that we [break immediately](https://github.com/grafana/grafana/blob/4ea7459b93d29044e96d9c23dbfc30bf7a99e58c/pkg/services/queryhistory/database.go#L294) whenever there is an error.

Reply:
Yes, you are correct. This can be removed.